src/Resources/config/enhancer.xml

```diff
@@ -12,7 +12,7 @@
                 public="false">
             <argument type="service" id="sonata.admin.pool" />
             <argument type="service" id="router" />
-            <tag name="cmf_resource.description.enhancer" alias="sonata_admin" />
+            <tag name="cmf_resource.description.enhancer" alias="sonata_phpcr_admin_integration" />
```

Comment:
As the master of resource-bundle still has that alias i got an collision. And when having a orm version with an description enhancer, we will have a collision too.